src/Components/ProgressBar.jsx

```diff
@@ -1,34 +1,36 @@
 import "../CSS/Calculator.css";
 
-const ProgressBar = ({ currentSectionName, currentSection, totalSections }) => {
+const ProgressBar = ({ currentSectionName, totalSections }) => {
   let sectionName;
-  let barWidthCurrent = `${Math.floor(
-    (currentSection / totalSections) * 100
-  )}%`;
-  let barWidthActive = `${Math.floor(
-    ((currentSection - 1) / totalSections) * 100
-  )}%`;
-  let stepArr = [];
-  let stepWidth = `${100 / totalSections - 4.5}%`;
+  let currentSection;
+  let stepWidth = `29%`;
+  let stepBackground = "#e5e5e5";
```

Comment:
can we add these to our `constants.css` file so that we can refer to the variable names here?